src/btree/bt_vrfy.c

```diff
@@ -582,6 +583,115 @@ __verify_addr_ts(WT_SESSION_IMPL *session, WT_REF *ref, WT_CELL_UNPACK *unpack,
     return (0);
 }
 
+/*
+ * __verify_btree_id_with_meta --
+ *     With a given btree id, iterate through the metafile table and check if the btree id exists in
+ *     any table. If exists, we place the table URI into a variable Otherwise should return
+ *     WT_NOTFOUND.
+ */
+static int
+__verify_btree_id_with_meta(WT_SESSION_IMPL *session, uint32_t btree_id, const char **uri)
+{
+    WT_CONFIG_ITEM id;
+    WT_CURSOR *meta_cursor;
+    WT_DECL_RET;
+    const char *meta_value;
+
+    WT_ERR(__wt_metadata_cursor(session, &meta_cursor));
```

Comment:
I'll add this since it's such a small change, is there a policy on out of scope changes being added to PR's ? Do we go for practicality of greater clarity? It makes our code much safer so I agree with adding it now.

Reply:
The policy? Got me... Ask @agorrod.